src/box/xrow_update_field.h

```diff
@@ -263,6 +263,28 @@ struct xrow_update_op {
 	bool is_for_root;
 };
 
+/**
+ * Return weather operation is scalar (all other then insert/delete/set).
+ */
+static inline bool
+xrow_update_op_is_scalar(struct xrow_update_op *op)
```

Comment:
Lets make the argument point at `const`.

Reply:
Done.